Sources/Nimble/Utils/Await.swift

```diff
@@ -264,7 +264,7 @@ internal class AwaitPromiseBuilder<T> {
                 #if swift(>=4.2)
                 _ = RunLoop.current.run(mode: .default, before: .distantFuture)
                 #else
-                _ = RunLoop.current.run(mode: .defaultRunLoopMode, before: .distantFuture)
+                _ = RunLoop.current.run(mode: RunLoop.Mode.default, before: .distantFuture)
```

Comment:
Swift 4.2 is already supported above.

```swift
                #if swift(>=4.2)
                _ = RunLoop.current.run(mode: .default, before: .distantFuture)
```

which had been addressed by https://github.com/Quick/Nimble/pull/549.

This breaks Swift 4.0/4.1 compatibility.